tests/postman/ElectionGuard API Tests.postman_collection.json

```diff
@@ -1052,15 +1103,280 @@
+											"tests[electionId] = electionId;\r",
+											""
+										],
+										"type": "text/javascript"
+									}
+								},
+								{
+									"listen": "prerequest",
+									"script": {
+										"exec": [
+											"const electionId = \"e2e-election-\" + pm.variables.get(\"e2e-run-id\");\r",
+											"pm.environment.set(\"e2e-election-id\", electionId);"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "PUT",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"election_id\": \"{{e2e-election-id}}\",\n    \"key_name\": \"e2e-key_ceremony_{{e2e-run-id}}\",\n    \"context\": {\n        \"commitment_hash\": \"{{e2e-commitment-hash}}\",\n        \"crypto_base_hash\": \"{{e2e-crypto-base-hash}}\",\n        \"crypto_extended_base_hash\": \"{{e2e-crypto-extended-base-hash}}\",\n        \"elgamal_public_key\": \"{{e2e-elgamal-public-key}}\",\n        \"manifest_hash\": \"{{e2e-manifest-hash}}\",\n        \"number_of_guardians\": 2,\n        \"quorum\": 2\n    }\n}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{mediator-url}}/api/{{version}}/election",
+									"host": [
+										"{{mediator-url}}"
+									],
+									"path": [
+										"api",
+										"{{version}}",
+										"election"
+									]
+								}
+							},
+							"response": []
+						}
+					]
 				},
 				{
-					"name": "3. Create Joint Key",
-					"item": []
+					"name": "5. Upload Ballots",
+					"item": [
+						{
+							"name": "5.1 Encrypt Ballots",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"Status code is 200\", function () {",
+											"    pm.response.to.have.status(200);",
+											"",
+											"    var jsonData = pm.response.json();",
+											"    const encryptedBallots = JSON.stringify(jsonData.encrypted_ballots);",
+											"    pm.environment.set('e2e-encrypted-ballots', encryptedBallots);",
+											"});",
+											"",
+											"pm.test(\"Ballots are returned in the same order as the request\", function () {",
+											"    const requestJson = JSON.parse(pm.request.body.raw)",
+											"    const responseJson = pm.response.json();",
+											"    pm.expect(responseJson.encrypted_ballots.length).to.eql(requestJson.ballots.length)",
+											"    for (let i = 0; i < requestJson.ballots.length; i++) {",
+											"        requestBallot = requestJson.ballots[i]",
+											"        responseBallot = responseJson.encrypted_ballots[i]",
+											"        pm.expect(responseBallot.object_id).to.eql(requestBallot.object_id)",
+											"    }",
+											"});",
+											"",
+											"pm.test(\"Ballots use proper code seed hashes\", function () {",
```

Comment:
I inadvertently copied this test over from something in the `ElectionGuard Web API` collection, but it was broken so I spent some time here figuring out what it was trying to say and converting it into the current structure.  I learned a lot in the process.